src/p5.dance.interpreted.js

```diff
@@ -56,25 +65,57 @@ function runUserSetup() {
   });
 }
 
+/**
+ * @param {Object} events - An object where each key is an event type. Each value
+ *  is another object, where the keys represeent the param of the event to be run
+ */
 function runUserEvents(events) {
-  var currentEvents = {};
+  // We have three separate event queues.
+  // First we run every N seconds/measures events
+  // Then we run after N seconds/measures events
```

Comment:
Sounds like this is really going to be "at", not "after", and even if we can't change the user-facing block, it's probably worth having the right nomenclature in the code.

Reply:
I'm tempted to leave this as-is because it's correct for the _current_ state of the world.  If we make changes to after/at, we should update this comment at that time.